flow/core/params.py

```diff
@@ -270,6 +276,27 @@ def add(self,
         lane_change_params : flow.core.params.SumoLaneChangeParams
             Params object specifying attributes for Sumo lane changing model.
         """
+
+        """# determine which energy model to use
+        if "prius" in veh_id:
+            self.energy_model = PriusEnergy()
+        elif "tacoma" in veh_id:
+            self.energy_model = TacomaEnergy
+        elif "ev" in veh_id:
+            self.energy_model = PDMElectric()
+        else:
+            self.energy_model = PDMCombustionEngine()
+            """
+
+        if energy_model is "prius":
+            self.energy_model = PriusEnergy(kernel, 0.9)
+        elif energy_model is "tacoma":
+            self.energy_model = TacomaEnergy()
+        elif energy_model is "ev":
+            self.energy_model = PDMElectric()
+        else:
+            self.energy_model = PDMCombustionEngine()
+   
```

Comment:
i don't think we need this anymore. follow the examples below for acceleration_controller (see `self.type_parameters`). also for reference, notice how controller classes are provided in `vehicles.add()`:
```
vehicles.add(
    "human",
    num_vehicles=0,
    lane_change_params=SumoLaneChangeParams(
        lane_change_mode="strategic",
    ),
    acceleration_controller=(IDMController, {
        "a": 1.3,
        "b": 2.0,
        "noise": 0.3,
    }),
    routing_controller=(I210Router, {}) if ON_RAMP else None,
)
```